packages/baml/src/services/baml-service.ts

```
@@ -67,6 +70,124 @@ export async function runAnthropicChat(message: string): Promise<string> {
+          request.data.patient_data,
+          request.data.treatment_options
+        );
+      
+      case "research-synthesis":
+        return await b.ResearchSynthesis(
+          request.data.papers,
+          request.data.research_question
+        );
+      
+      default:
+        throw new Error(`Unknown insight type: ${(request as any).type}`);
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)